spec/spec_helper.rb

```diff
@@ -3,6 +3,9 @@
 require 'rspec'
 require 'pivotal-to-trello'
 
+# These API Tokens are for a user with just one Public Sample Project
+PT_USER_1 = { token: 'ab4c5895f57995bb7547986eacf91160'}
```

Comment:
This doesn't really need to be a hash at all. the tracker-api tests had more information about the user but we're just using token

Reply:
This is a real account? I normally prefer to mock remote APIs, but if you're happy to include the token I'm happy to try it and see how long it lasts 👍